src/main/scala/cromwell/engine/backend/jes/Run.scala

```diff
@@ -57,10 +59,15 @@ object Run  {
   }
 }
 
-case class Run(name: String, pipeline: Pipeline, tag: String) {
+case class Run(jesId: String, pipeline: Pipeline, tag: String) {
+
+  val dataAccess = DataAccess()
```

Comment:
OK, I'll try to rework all this data access stuff into a singleton (except for those tests)